scripts/devops_tasks/common_tasks.py

```diff
@@ -461,3 +462,9 @@ def get_installed_packages(paths = None):
     ws = WorkingSet(paths) if paths else working_set  
     return ["{0}=={1}".format(p.project_name, p.version) for p in ws]
 
+def get_package_properties(setup_py_path):
```

Comment:
So it's a new SDK if the package is azure-core, azure-mgmt-core, or if any of the requirements are one of those? 

What?

Reply:
For reference we are following https://github.com/Azure/azure-sdk/issues/1261#issuecomment-691327942. The idea is we needed a way to detect if a package is track 2 (aka new).